nbed/localizers/spade.py

```diff
@@ -151,7 +151,7 @@ def localize_virtual(self, embedded_scf: StreamObject) -> StreamObject:
                 [embedded_scf.mo_coeff[i][:, occ[i] == 0] for i in [0, 1]]
             )
 
-        logger.debug(f"N effective viruals: {effective_virt.shape}")
+        logger.debug(f"N effective virtuals: {effective_virt.shape}")
```

Comment:
instead of number of effective virtuals, does this not give the size of the `C_vir` matrix? - number of effective virtuals is `effective_virt.shape[1]`?